src/commands/append.json

```diff
@@ -0,0 +1,51 @@
+{
+    "APPEND": {
+        "summary": "Append a value to a key",
+        "complexity": "O(1). The amortized time complexity is O(1) assuming the appended value is small and the already present value is of any size, since the dynamic string library used by Redis will double the free space available on every reallocation.",
+        "group": "string",
+        "since": "2.0.0",
+        "arity": 3,
+        "function": "appendCommand",
+        "command_flags": [
+            "write",
+            "denyoom",
+            "fast"
+        ],
+        "acl_categories": [
+            "write",
+            "string",
+            "fast"
+        ],
```

Comment:
maybe we do want to keep some of the acl categories implicit (i.e. automatically added by the command flags)?
it may be a bit confusing, but at least there's no redundant info and a chance for a conflict.

Reply:
it was generated from COMMAND's reply, i guess we can reverse-engineer it and omit acl categories that were implicitly added

sounds good?

Reply:
done

Reply:
so now in some sense the JSON files are different than what COMMAND command generates, right?

Reply:
right

Reply:
so maybe we should make them even more different, to make sure no one uses them by mistake instead of getting them from redis.
first, we can change `loading` back to `ok-loading`, but maybe we can do something even more drastic to guarantee no one uses them?